go/vt/vtctl/vdiff2.go

```diff
@@ -56,6 +56,7 @@ func commandVDiff2(ctx context.Context, wr *wrangler.Wrangler, subFlags *pflag.F
 	tabletTypes := subFlags.String("tablet_types", "in_order:RDONLY,REPLICA,PRIMARY", "Tablet types for source (PRIMARY is always used on target)")
 
 	debugQuery := subFlags.Bool("debug_query", false, "Adds a mysql query to the report that can be used for further debugging")
+	maxVDiffReportRows := subFlags.Uint64("max-vdiff-report-rows", 10, "Maximum number of row differences to report (0 for all differences). NOTE: when increasing this value it is highly recommended to also specify --only_pks=true")
```

Comment:
The command is vdiff so using vdiff in the flag name is redundant. Also note that `--only_pks` is good (no need to use =true).